### PR TITLE
🧹 Cleanup `Channel` model fields

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/channels/models.py [hunk 1]
```diff
@@ -391,20 +353,20 @@ class Channel(LegacyUUIDMixin, TembaModel, DependencyMixin):
...
+    os = models.CharField(max_length=255, null=True, blank=True)
+    last_seen = models.DateTimeField(null=True)
 
-    tps = models.IntegerField(
-        verbose_name=_("Maximum Transactions per Second"),
-        null=True,
-        help_text=_("The max number of messages that will be sent per second"),
-    )
+    # TODO drop
+    bod = models.TextField(null=True)
```
Comment (rowanseymour) [inline temba/channels/models.py:369]: will do in separate PR because dropping columns requires faking migrations

File: temba/channels/models.py [hunk 2]
```diff
@@ -391,20 +353,20 @@ class Channel(LegacyUUIDMixin, TembaModel, DependencyMixin):
...
     parent = models.ForeignKey("self", on_delete=models.PROTECT, null=True)
+    tps = models.IntegerField(null=True)
 
-    bod = models.TextField(null=True)
+    # Android relayer specific fields
+    claim_code = models.CharField(max_length=16, blank=True, null=True, unique=True)
+    secret = models.CharField(max_length=64, blank=True, null=True, unique=True)
+    device = models.CharField(max_length=255, null=True, blank=True)
+    os = models.CharField(max_length=255, null=True, blank=True)
+    last_seen = models.DateTimeField(null=True)
```
Comment (rowanseymour) [inline temba/channels/models.py:366]: we can leave this null for non-Android channels which don't use it

**Review by ericnewcomer — APPROVED** [review]
